### ISSUE TITLE
Error messages should mention affected topic inline

### ISSUE THREAD

**Issue by xephon2** [dita-ot/dita-ot#2638]
I slurp the DITA-OT Ant log through a filterchain and keep only lines containing `[FATAL]`, `[ERROR]`, `[WARNING]` and `[INFO]`. Unfortunately, sometimes important context information gets lost.

It would be nice, if the affected topics would be mentioned inline in all error messages, because messages without context are useless:

```
[topicpull] [DOTX028E][ERROR]: Link or cross reference must contain a valid href or keyref attribute; no link target is specified.
[clean-map] [DOTX072I][INFO]: Ignoring navtitle within topicgroup.
```

**Comment by jelovirt** [comment]
The topic URI and line/column information is available in some messages. Not all logging code has been modified to pass those through yet.

**Comment by xephon2** [comment]
Will be fixed with #2724
